File: src/EstructuraBE/Modelos/RespuestaPQRS.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;
using System.Collections.Generic;

namespace juntacomunal.EstructuraBE.Modelos{
    public class RespuestaPQRS
    {
        [Key]
        public string respuestaId { get; set; }
        [Required]
        public string tituloRespuesta { get; set; }
        [Required]
        public string descripcionRespuesta { get; set; } 
        [Required]
        public string autorRespuesta { get; set; }    
        [Required]
        public string estadoRespuesta { get; set; }   
        public string fechaCreacion { get; set; }
        public string destinatarioRespuesta { get; set; }
    }
}

File: src/app/core/Modelos/respuesta-pqrs.ts
import { PQRS } from './pqrs';

export class RespuestaPQRS {
    respuestaId:string | undefined;
    tituloRespuesta:string | undefined;
    descripcionRespuesta:string | undefined;
    autorRespuesta:string | undefined;
    estadoRespuesta:string | undefined;
    fechaCreacion: any | undefined;
    destinatarioRespuesta:string | undefined;
    pqrs: PQRS | undefined;
}
